src/overlays/actors/ovl_Obj_Blockstop/z_obj_blockstop.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_obj_blockstop.c
  * Overlay: ovl_Obj_Blockstop
- * Description:
+ * Description: Signals that a block has fallen in a hole and can't be moved
```

Comment:
Signals how? Via audio cue? (I'm on the hunt for all actors that are audio-based)

Reply:
It doesn't appear to be responsible for audio. It *goes* appear to be responsible for playing a cutscene though. It's only in two rooms of Snowhead Temple.